src/components/OurTeam/index.css

```diff
@@ -0,0 +1,11 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.teamMain {
```

Comment:
Whats does team main means, can you give it more meaningful name

Reply:
replaced with teamSection